### PR TITLE
Deployment tooling update - bin/eb-env-sync

### PR DESCRIPTION
**Pull request by indigobravo**
The bin/eb-env-sync script has been updated to handle multi-region
deployments.

### CODE REVIEW

File: bin/eb-env-sync [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+    echo "  REGION         the target AWS region"
     echo
-    echo "The environment configuration will be assumed to live in an "
+    echo "us-west-1 environment configuration will be assumed to live in an "
     echo "environment manifest at <APP>/env-<ENV>.yml relative to the working "
     echo "directory."
+    echo
+    echo "ca-central-1 environment configuration will be assumed to live in an "
+    echo "environment manifest at <APP>/env-<ENV>-ca.yml relative to the working "
+    echo "directory."
```
Comment (indigobravo) [inline bin/eb-env-sync:19]: Help message has been updated.

File: bin/eb-env-sync [hunk 2]
```diff
@@ -24,7 +29,7 @@ status () {
     echo "--->" "$@" >&2
 }
 
-if [ "$#" -ne 2 ]; then
+if [ "$#" -ne 3 ]; then
```
Comment (indigobravo) [inline bin/eb-env-sync:32]: Mandatory parameters has been increased to 3 now that we are passing in the REGION.

File: bin/eb-env-sync [hunk 3]
```diff
@@ -35,7 +40,15 @@ PATH="$(dirname "$0"):${PATH}"
...
-MANIFEST="${APP}/env-${ENV}.yml"
+REGION=$3
+
+if [ $REGION = 'us-west-1' ]; then
+    MANIFEST="${APP}/env-${ENV}.yml"
+fi
+
+if [ $REGION = 'ca-central-1' ]; then
+    MANIFEST="${APP}/env-${ENV}-${REGION:0:2}.yml"
+fi
```
Comment (indigobravo) [inline bin/eb-env-sync:51]: Conditionally set `MANIFEST` based on the REGION parameter

File: bin/eb-env-sync [hunk 4]
```diff
@@ -70,6 +83,7 @@ status "updating application resource lifecycle settings"
 aws elasticbeanstalk update-application-resource-lifecycle \
     --application-name "$APP" \
     --resource-lifecycle-config file://resource-lifecycle.json
+    --region $REGION
```
Comment (indigobravo) [inline bin/eb-env-sync:86]: Added `--region` flag to `aws` command.

File: bin/eb-env-sync [hunk 5]
```diff
@@ -78,3 +92,4 @@ aws elasticbeanstalk update-environment \
     --environment-name "${APP}-${ENV}" \
     --platform-arn "$EB_PLATFORM_ARN" \
     --option-settings file://"$OPTION_SETTINGS_FILE"
+    --region $REGION
```
Comment (indigobravo) [inline bin/eb-env-sync:95]: Added `--region` flag to `aws` command.

**Review by robertknight — APPROVED** [review]

**Pull request merged**